### PR TITLE
fix: migrate to tunnistus and fix hardcoded profile settings

### PR DESCRIPTION
**Pull request by voneiden**
Refs: PSA-22, PSA-37

### CODE REVIEW

**Comment by terovirtanen** [comment]
**PYSAKOINNIN-SAHK-ASIOINTI-UI branch is deployed to platta: https://pysakoinnin-sahk-asiointi-pr174-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://pysakoinnin-sahk-asiointi-pr174-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: .env.test.local [hunk 1]
```diff
@@ -1,4 +1,3 @@
 REACT_APP_API_URL="http://localhost:8000"
-REACT_APP_API_BACKEND_TOKEN_URL="http://localhost:8000"
-REACT_APP_PROFILE_AUDIENCE="http://localhost:8000"
-REACT_APP_API_BACKEND_PERMISSION="https://api.hel.fi/auth/helsinkiprofile"
\ No newline at end of file
+REACT_APP_API_CLIENT_ID="http://localhost:8000"
+REACT_APP_PROFILE_API_CLIENT_ID="https://api.hel.fi/auth/helsinkiprofile"
```
Comment (tuomas777) [inline .env.test.local:3]: I guess a new line could be added here 🙂

**Review by tuomas777 — APPROVED** [review]
LGTM 👍 🅿️

File: .env.test.local (hunk 1 above)
Comment (voneiden) [inline .env.test.local:3]:
```suggestion
REACT_APP_PROFILE_API_CLIENT_ID="https://api.hel.fi/auth/helsinkiprofile"

```

**Comment by terovirtanen** [comment]
**PYSAKOINNIN-SAHK-ASIOINTI-UI branch is deployed to platta: https://pysakoinnin-sahk-asiointi-pr174-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://pysakoinnin-sahk-asiointi-pr174-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**